x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/last_first_seen/index.ts

```diff
@@ -11,18 +11,18 @@ import type { IEsSearchResponse } from '@kbn/data-plugin/common';
 import type {
   FactoryQueryTypes,
   FirstLastSeenStrategyResponse,
-  FirstLastSeenRequestOptions,
 } from '../../../../../common/search_strategy/security_solution';
 import { FirstLastSeenQuery } from '../../../../../common/search_strategy/security_solution';
 
 import { inspectStringifyObject } from '../../../../utils/build_query';
 import type { SecuritySolutionFactory } from '../types';
 import { buildFirstOrLastSeenQuery } from './query.first_or_last_seen.dsl';
+import { parseOptions } from './parse_options';
 
 export const firstOrLastSeen: SecuritySolutionFactory<typeof FirstLastSeenQuery> = {
-  buildDsl: (options: FirstLastSeenRequestOptions) => buildFirstOrLastSeenQuery(options),
+  buildDsl: (options: unknown) => buildFirstOrLastSeenQuery(options),
```

Comment:
Does it make sense to use below type here instead of unknown?

https://github.com/elastic/kibana/blob/48d73c315d56033b7d88a93a11dd68075f3b98cb/x-pack/plugins/security_solution/common/api/search_strategy/first_seen_last_seen/first_seen_last_seen.ts#L22-L24